matrixprofile/analyze.py

```diff
@@ -209,6 +212,10 @@ def analyze(ts, query=None, windows=None, sample_pct=1.0, threshold=0.98, n_jobs
         window(s) is given.
     n_jobs : int, Default = 1
         Number of cpu cores to use.
+    preprocessing_kwargs : dict, default = None
```

Comment:
Please be more verbose about what the preprocessing kwargs are. This will make it easier for the end-users to know what to do via the API documentation.